go/libkb/config.go

```diff
@@ -672,6 +672,10 @@ func (f JSONConfigFile) GetUpdateURL() string {
 	return s
 }
 
+func (f JSONConfigFile) IsAdmin() (bool, bool) {
```

Comment:
Second bool is if a value was present at all (if set)... 

For example you might want the default value to be true (if not set) in which case you check if not set and return true.

I'm following the convention of returning bools this way in this class for consistency.